content/podcast/2024/ep29_Nextflow-2023-2024.md

```diff
@@ -0,0 +1,53 @@
+title=Nextflow 2023 - 2024
+episode=29
+description=We recap of some of the biggest features to be added to Nextflow in 2023 and take a look at some of the things coming in 2024.
+date=2024-01-23
+type=podcast
+subtype=Community highlights
+youtubeid=T7YpCcqFQpg
+image=img/podcast_ep29.png
+tags=nextflow,opensource,community,nf-core
```

Comment:
consider deleting community,nf-core tags?

Reply:
Yup, was just copy-paste and didn't spend much time thinking about this
```suggestion
tags=nextflow,opensource
```